src/Microsoft.ML.Transforms/OneHotEncoding.cs

```diff
@@ -165,7 +165,7 @@ internal OneHotEncodingTransformer(ValueToKeyMappingEstimator term, IEstimator<I
 
         public IDataView Transform(IDataView input) => _transformer.Transform(input);
 
-        public void Save(ModelSaveContext ctx) => _transformer.Save(ctx);
+        void ICanSaveModel.Save(ModelSaveContext ctx) => (_transformer as ICanSaveModel).Save(ctx);
```

Comment:
>_transformer as ICanSaveModel [](start = 58, length = 29)

Do not use `as` style casts unless you are entertaining the possibility that it might not be this thing. We know for a fact it must be this thing, so `()` style casts are appropriate.

Reply:
Thank you, I didn't know!

---
In reply to: [254489054](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254489054) [](ancestors = 254489054)